Duplicati/Library/Common/IO/SystemIOWindows.cs

```diff
@@ -487,7 +487,11 @@ IFileEntry FileEntryAlphaFS(string p)
                 objs.Add(new FileSystemAccess((FileSystemAccessRule)f));
 
             dict["win-ext:accessrules"] = SerializeObject(objs);
-            dict["win-ext:accessrulesprotected"] = rules.AreAccessRulesProtected.ToString();
+
+            if (rules.AreAccessRulesProtected)
+            {
+                dict["win-ext:accessrulesprotected"] = rules.AreAccessRulesProtected.ToString();
+            }

```

Comment:
Can this be simplified to 
```
if (rules.AreAccessRulesProtected)
{
    dict["win-ext:accessrulesprotected"] = "True";
}
```
or alternatively without the `if`
```
dict["win-ext:accessrulesprotected"] = rules.AreAccessRulesProtected.ToString();
```

Reply:
Your first suggestion is fine with me if you like that better.

Second one is how code is right now... we want to avoid listing this property in metadata when the value is False

Reply:
> Second one is how code is right now... we want to avoid listing this property in metadata when the value is False

Oops, not sure what I was thinking here. 

> Second one is how code is right now... we want to avoid listing this property in metadata when the value is False

I think I understand the issue now.  Is this simply a workaround that can be removed once the next beta is released?  If so, can we add a comment in the code explaining why we're doing this, and also create a ticket to clean this up after the next beta is released?

Reply:
Sorry, I'm probably not explaining the issue very well.

This change should stay indefinitely.  Versions prior to 2.0.5.101 did not store `win-ext:accessrulesprotected` at all - it was added very recently.  But when I did add support for it, I had Duplicati store it for every single file with a value of True or False.  This had a negative side effect of making Duplicati think the metadata changed for every single file (the first time you ran a backup after upgrading), and caused it to reprocess all file contents.  An unnecessary - and potentially lengthy - operation.

If we change it so that `win-ext:accessrulesprotected` is only stored when True, then metadata will appear the same as how Duplicati 2.0.5.1 (and older) calculated it for files where inheritance is NOT broken (which is the vast majority of files). 

Not sure if that explanation is any better...

Reply:
I see.  This is still probably worth explaining in the code with a comment, since at a quick glance it's not clear why "False" is different than the option being unset.

Reply:
Sure, I'll put in a comment and also implement your other suggestion....thanks